.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,25 @@
+name: Open release pr
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on: workflow_dispatch
```

Comment:
This is usually `on: { push: { branches: [main] } }` so that the workflow keeps a live PR up to date with changes. This way we won't have to trigger the workflow manually to create a release.

Reply:
IMO trigging the workflow itself is pretty easy and give us more control, sometimes we might want to release multiple commits (when we know they will be merged pretty soon), or we don't want to release commits that just fix the CI.

Reply:
The release PRs are merged manually.

I have this set up on [robjtede/inspect-cert-chain](https://github.com/robjtede/inspect-cert-chain). See [#165](https://github.com/robjtede/inspect-cert-chain/pull/165) for example.

Reply:
https://github.com/Nullus157/async-compression/pull/280